src/api/.haml-lint_todo.yml

```diff
@@ -14,4 +14,6 @@ linters:
 
   # Offense count: 105
   SetPagetitleInView:
-    enabled: false
+    enabled: true
+    exclude:
+      - 'app/views/webui/users/new.html.haml'
```

Comment:
This `SetPagetitleInView` should be removed from this file, and inserted into the `src/api/.haml-lint.yml` file.